### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: data_models.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+require 'data_mapper'
+
+if ENV['RACK_ENV'] != 'production'
+  require 'dotenv'
+  Dotenv.load('.env')
+end
+
+DataMapper.setup(:default, ENV['DATABASE_URL'])
```
Comment (ghost) [inline data_models.rb:11]: @openspectrum: I got this from the can-i-eat-here app. Why store the database URL in `.env`?

Comment (jfarmer) [inline data_models.rb:11]: You use a different database in development, testing, and production.  This is the name of the ENV var heroku uses

## 

Jesse Farmer (via iPhone)

http://codeunion.io · @jfarmer

jesse@codeunion.io · 773.319.9355

File: data_models.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+class User
+  include DataMapper::Resource
+
+  property :id,         Serial
+  property :email,      String, :required => true, :format => :email_address
+  property :nick_name,  String, :required => true, :unique => true
+  property :password,   String, :required => true, :length => 6..20
+end
```
Comment (tannerwelsh) [inline data_models.rb:25]: This looks like a good start. Eventually you'll want to encrypt passwords and store the password hash (not plain text).

Also, be aware that this schema design for `User` allows for multiple users to have the same email address, which could cause some confusion down the line. Probably better to add a uniqueness constraint to your email field.

Comment (ghost) [inline data_models.rb:25]: :white_check_mark: :+1:

**Pull request closed without merging**